test/func_network.py

```diff
@@ -85,9 +86,12 @@ def network_pktdriver_mtcp(self, driver):
 $_tapdev = "tap0"
 """, timeout=30)
 
-    p.join()
-    sleep(5)
```

Comment:
I think this sleep is needed.
Even the successful `join()` wasn't
enough to fix the TAP shutdown problem.